src/infrastructure/acknowlegement.rs

```diff
@@ -3,7 +3,7 @@ use crate::infrastructure::{ExternalAcks, LocalAckRecord};
 /// Type responsible for handling the acknowledgement of packets.
 pub struct AcknowledgementHandler {
     waiting_packets: LocalAckRecord,
-    their_acks: ExternalAcks,
+    our_acks: ExternalAcks,
```

Comment:
I find the name `our_ackets` a bit confusing because now it seems like those acks are from our self but they are from 1 'them/their'

Reply:
The acks *are* from ourself! They are our acknowledgements of foreign packet receipt

Reply:
Thanks for clarifying your vision, let's leave it then.